front/src/modules/settings/data-model/objects/SettingsObjectCoverImage.tsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { IconX } from '@/ui/display/icon';
+import { LightIconButton } from '@/ui/input/button/components/LightIconButton';
+import { AnimatedFadeOut } from '@/ui/utilities/animation/components/AnimatedFadeOut';
+import { cookieStorage } from '~/utils/cookie-storage';
+
+import CoverImage from '../assets/build-your-business-logic.jpg';
+
+const StyledCoverImageContainer = styled.div`
+  border: 1px solid ${({ theme }) => theme.border.color.medium};
+  border-radius: ${({ theme }) => theme.border.radius.md};
+  height: 153px;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
```

Comment:
is this position necessary?

Reply:
Yes, otherwise the absolute positioning of the child is incorrect